docs-chef-io/content/server/upgrades.md

```diff
@@ -44,6 +44,17 @@ Chef Infra Server 14 moved from Solr to Elasticsearch as its search index.
 
 The Chef Infra Server 14 upgrade does not automatically reindex existing external Elasticsearch installations.
 
+### Upgrading to 14.8.x
+
+Chef Infra Server 14.8 upgraded postgresql from 9.6 to 13.3. There are a couple of extra steps that need to be considered depending on the scenario. 
```

Comment:
Can we add information here about what the expected downtime will be? And also point them to additional system requirements before getting started with the upgrade.